ompi/mca/bml/r2/Makefile.am

```diff
@@ -37,6 +38,9 @@ mcacomponentdir = $(ompilibdir)
 mcacomponent_LTLIBRARIES = $(component_install)
 mca_bml_r2_la_SOURCES = $(r2_sources)
 mca_bml_r2_la_LDFLAGS = -module -avoid-version
+mca_bml_r2_la_LIBADD = $(top_builddir)/ompi/lib@OMPI_LIBMPI_NAME@.la \
+	$(top_builddir)/orte/lib@ORTE_LIB_PREFIX@open-rte.la \
```

Comment:
blah; First, we should find a way to macro-ize this, so there's only one thing in LIBADD.

But second, is a dependency on all three libraries the right thing here?  It means that a bump in the major version of any one of the three will result in a load failure, which I guess is fine?